ush/forecast_postdet.sh

```diff
@@ -693,6 +693,11 @@ MOM6_postdet() {
       ${NLN} "${COM_OCEAN_ANALYSIS}/${RUN}.t${cyc}z.ocninc.nc" "${DATA}/INPUT/mom6_increment.nc"
   fi
 
+  # GEFS perturbations
+  if (( 0${MEMDIR:3} > 0 )) && [[ "${ODA_INCUPD:-False}" == "True" ]]; then
+     ${NLN} "${COM_OCEAN_RESTART_PREV}/${sPDY}.${scyc}0000.mom6_increment.nc" "${DATA}/INPUT/mom6_increment.nc"
+  fi
+  
```

Comment:
We may need to wrap this in a `if [[ $RUN} == "gefs" ]]` block to ensure it does not interfere with the GFS ensemble.  That component is not ready yet, so we may just add a `TODO` for now and leave it at that.